src/Mod/Part/Gui/ViewProviderExt.cpp

```diff
@@ -951,7 +951,19 @@ void ViewProviderPartExt::updateVisual()
 
         // create or use the mesh on the data structure
         Standard_Real AngDeflectionRads = AngularDeflection.getValue() / 180.0 * M_PI;
+
+#if OCC_VERSION_HEX < 0x070500
         BRepMesh_IncrementalMesh(cShape, deflection, Standard_False, AngDeflectionRads, Standard_True);
+#else
+        IMeshTools_Parameters meshParams;
+        meshParams.Deflection = deflection;
+        meshParams.Relative = Standard_False;
+        meshParams.Angle = AngDeflectionRads;
+        meshParams.InParallel = Standard_True;
+        meshParams.AllowQualityDecrease = Standard_True;
+
+        BRepMesh_IncrementalMesh(cShape, meshParams);
+#endif

```

Comment:
If works, something like this seems cleaner.
```suggestion
        IMeshTools_Parameters meshParams;
        meshParams.Deflection = deflection;
        meshParams.Relative = Standard_False;
        meshParams.Angle = AngDeflectionRads;
        meshParams.InParallel = Standard_True;
#if OCC_VERSION_HEX >= 0x070500
        meshParams.AllowQualityDecrease = Standard_True;
#endif
        BRepMesh_IncrementalMesh(cShape, meshParams);
```

Reply:
It seems IMeshTools_Parameters was introduced in 7.4 and our minimum supported version is 7.3, so I think the PR is correct as is.

ref: https://dev.opencascade.org/doc/occt-7.4.0/overview/html/occt_dev_guides__upgrade.html

Reply:
Exactly, @adrianinsaval!

The `Ubuntu_20-04` Build uses OCC 7.3.0

`IMeshTools_Parameters` was introduced in OCC 7.4.0
https://old.opencascade.com/doc/occt-7.3.0/refman/html/class_b_rep_mesh___incremental_mesh.html
https://old.opencascade.com/doc/occt-7.4.0/refman/html/class_b_rep_mesh___incremental_mesh.html

`AllowQualityDecrease` was introduced in OCC 7.5.0
https://old.opencascade.com/doc/occt-7.4.0/refman/html/struct_i_mesh_tools___parameters.html
https://old.opencascade.com/doc/occt-7.5.0/refman/html/struct_i_mesh_tools___parameters.html